src/appleseed.python/studio/__init__.py

```diff
@@ -27,3 +27,4 @@
 #
 
 from _appleseedstudio import *
+from ui import *
```

Comment:
We don't need to import ui here. Because it imports Qt, it will fail for users without any Qt binding.